clients/java/src/test/java/io/camunda/zeebe/client/tenant/RemoveUserFromTenantTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.tenant;
+
+import static io.camunda.zeebe.client.impl.http.HttpClientFactory.REST_API_PATH;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.github.tomakehurst.wiremock.http.RequestMethod;
+import io.camunda.zeebe.client.api.command.ProblemException;
+import io.camunda.zeebe.client.protocol.rest.ProblemDetail;
+import io.camunda.zeebe.client.util.ClientRestTest;
+import io.camunda.zeebe.client.util.RestGatewayService;
+import org.junit.jupiter.api.Test;
+
+public class RemoveUserFromTenantTest extends ClientRestTest {
+
+  private static final long TENANT_KEY = 123L;
+  private static final long USER_KEY = 456L;
+
+  @Test
+  void shouldRemoveUserFromTenant() {
+    // when
+    client.newRemoveUserFromTenantCommand(TENANT_KEY).userKey(USER_KEY).send().join();
+
+    // then
+    final String requestPath = RestGatewayService.getLastRequest().getUrl();
+    final RequestMethod method = RestGatewayService.getLastRequest().getMethod();
+    assertThat(requestPath)
+        .isEqualTo(REST_API_PATH + "/tenants/" + TENANT_KEY + "/users/" + USER_KEY);
+    assertThat(method).isEqualTo(RequestMethod.DELETE);
+  }
+
+  @Test
+  void shouldRaiseExceptionOnNotFoundTenant() {
```

Comment:
Since these error tests all essentially test the same thing we could combine them in a parametrized tests.

Reply:
it makes sense, we can globally do this in a separate PR